src/components/ContactPolicy/ContactPolicy.test.js

```diff
@@ -31,7 +30,7 @@ describe('ContactPolicy component', () => {
     active: false,
     emailsAmountByInterval: 20,
     intervalInDays: 7,
-    excludedSubscribersLists: [],
+    excludedSubscribersLists,
```

Comment:
How about a test to simulate having one excluded list, and check the tag is created?

Reply:
As we talked by chat this is already tested inside the component for each CloudTagField, so it's ok as it is.